src/edu/ucsb/nceas/metacat/dataone/D1NodeService.java

```diff
@@ -259,7 +275,8 @@ public Identifier delete(Session session, Identifier pid)
                 HazelcastService.getInstance().getIdentifiers().remove(pid);
                 sysMeta.setArchived(true);
```

Comment:
Setting the archive true is the old way to delete the solr index. Since we have a new way to delete solr index, we can remove this line.
9701fa31074ed6b90e672ec94cd4178b81ed3677